exchanges/order/order_test.go

```diff
@@ -1906,20 +1536,17 @@ func TestDeriveCancel(t *testing.T) {
 		AssetType:     asset.Futures,
 	}
 	cancel, err := o.DeriveCancel()
-	if !errors.Is(err, nil) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
-	if cancel.Exchange != "wow" ||
+	require.NoError(t, err)
+
+	assert.False(t, cancel.Exchange != "wow" ||
```

Comment:
Can expand thse bad boys and assertify them individually like how you've done it elsewhere in this PR